alpine/Dockerfile

```diff
@@ -19,28 +18,29 @@ ARG LOCAL_KONG_PACKAGE=empty-kong.tar.gz
 COPY $LOCAL_KONG_PACKAGE /tmp/kong.tar.gz
 
 RUN if [ "$ASSET_LOCATION" = "local" ] ; then exit 0 ; fi; \
-    . /tmp/build.sh
+    apk add --no-cache --virtual .build-deps curl wget tar ca-certificates && \
+    . /tmp/build.sh && \
+    apk del .build-deps
 	
 RUN mkdir /kong; \
-	tar -C /kong -xzf /tmp/kong.tar.gz
-
-FROM alpine:3.11
-
-LABEL maintainer="Kong <support@konghq.com>"
-
-COPY --from=downloader /kong/usr/local /usr/local
-COPY --from=downloader /kong/etc /etc
+	tar -C /kong -xzf /tmp/kong.tar.gz && \
```

Comment:
Obtaining the package in a separate step (the copy at the top) will cause the final image to have one unneeded extra layer, increasing the image size, compared to old versions of the Dockerfile.

Reply:
The `COPY` command is copying an empty file which results in a 0 byte noop. It adds a layer but I don't think there's anyways to `IF` a `COPY` command currently